tests/testthat/test-extractors.R

```diff
@@ -3,12 +3,12 @@ test_that("tweets_data works", {
 
   ## get data on most recent tweet from user(s)
   tweets <- tweets_data(jack)
-  expect_s3_class(tweets, "tbl_df")
-  expect_true("status_id" %in% names(tweets))
+  expect_s3_class(tweets, "data.frame")
```

Comment:
Is it a deliberate change to go from tibbles to data.frames? data.frames are not great at printing list-columns.

Reply:
Yes, I was getting some warnings that I reported here: https://github.com/tidyverse/tibble/issues/898 and I decided to go back to data.frames to avoid users (and the package) getting that warning. Perhaps it should be transformed to a tibble before returning to the users via tweets_data...

Reply:
Yeah. I think tibble should eliminate that warning — I'll raise this internally.